project/AppDependencies.scala

```diff
@@ -6,11 +6,11 @@ object AppDependencies {
   val compile: Seq[ModuleID] = Seq(
     ws,
     "uk.gov.hmrc" %% "bootstrap-play-26" % "1.7.0",
-    "uk.gov.hmrc" %% "domain" % "5.6.0-play-26",
+    "uk.gov.hmrc" %% "domain" % "5.8.0-play-26",
     "uk.gov.hmrc" %% "play-partials" % "6.10.0-play-26",
-    "uk.gov.hmrc" %% "play-ui" % "8.8.0-play-26",
+    "uk.gov.hmrc" %% "play-ui" % "8.9.0-play-26",
     "uk.gov.hmrc" %% "http-caching-client" % "9.0.0-play-26",
-    "uk.gov.hmrc" %% "auth-client" % "3.0.0-play-26",
+    "uk.gov.hmrc" %% "auth-client" % "3.0.0-play-27",
```

Comment:
You cannot just upgrade to this version - it involves a complete Play 27 upgrade. We currently are working on Play 26